db/migrations/20170329150700_create_client_applications.rb

```diff
@@ -0,0 +1,21 @@
+Sequel.migration do
+  up do
+    create_table :client_applications do
+      primary_key :id
+      String :name, null: false
+      String :client_id, null: false
+      String :client_secret, null: false
+      Boolean :in_house_app, null: false, default: false
+      Integer :user_id, null: false
```

Comment:
We should use `foreign_key` for all associates which means postgres won't let a value go in that isn't a valid row in the user table - [sequel docs](http://sequel.jeremyevans.net/rdoc/files/doc/migration_rdoc.html#label-What+to+put+in+your+migration-27s+down+block)


```
foreign_key :user_id, :users
```